### ISSUE TITLE
Design adjustments

### ISSUE THREAD

**Issue by diana-lakatos** [Platform-OS/platformos-documentation#734]
We would like to have some minor design adjustments on the documentation site before the end of October. They would be small adjustments, no big change:
- link colours and hover states
- alert white spaces
- adding some more colour to the home page (if needed)

**Comment by diana-lakatos** [comment]
Update: 

- links fixed 
- found a way to fix alerts, assigning to myself
- we will think about adding color later

**Comment by diana-lakatos** [comment]
Status: Once the alert-fixes PR is merged, we close this issue, as the only remaining task (add color to homepage) is postponed.
